docs/mesos/DCOS.md

```diff
@@ -0,0 +1,134 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Installing Calico in DCOS
+This guide provides instructions for installing Calico and its
+prerequisites on a Mesos DCOS cluster.
+
+##Prerequisites
+
+### Mesos Master and Agents
+
+You will need to be running a cluster with at least one Mesos master
+and and at least one Mesos agent (though we recommend having two or
+more agents).
+
+We currently support Mesos version `v0.28.0` and on for use with DCOS.
+Additionally, we support Centos7 as an operating system for use with
+the Unified Containerizer and Docker Containerizer, though additional
+OS's *may* work (though have not been fully tested!) with the Docker
+Containerizer.
+
+### Etcd
+Calico uses etcd as its datastore.
+
+To install etcd:
+ - Go to the **Universe** tab in your DCOS UI.
+ - Find the **etcd** package
+ - Click the **Install** button
+ - (Optional) choose **Advanced Installation** from
+the pop-up window.
+	- In non-production environments used for testing, set all of
+      the **mem** options to use 50MB or lower (this includes
+      `mem` and `mem-limit`).
+ - Choose **Review and Install**
+ - Double-check configuration then click **Install**.
+
+You can check the status of etcd by visiting the **Services**
+tab. You can install Calico once etcd appears with a status
+of **`Healthy`** (note that this could take a few minutes).
+
+## Install Calico
+You can easily install Calico from within the DCOS UI.
+
+### Get Calico Package
+You will need to link the Calico package from our git
+repository before you can install it.  To do this:
+
+ - Click the **System** tab
+ - Select **Add Repository** in the upper right
+ - Enter a name such as "Calico" for the package
+ - Paste the following git zip archive as the package link:
+   `https://github.com/robbrockbank/universe/archive/calico-universe.zip`
+
+### Install Calico Package
+You can now install the Calico package from the **Universe**
+tab. To start the install:
+ - Go to the **Universe** tab
+ - Find the Calico package
+ - Click the **Install** button then choose **Advanced
+   Installation** in the pop-up window.
+
+![alt tag](../images/mesos/dcos-calico-package-install.png)
+
+##### At this point you have a few options:
+
+In non-production environments used for testing, 
+you can set each memory limit option to have a
```

Comment:
although add a note that Docker is old (see https://github.com/dcos/dcos-vagrant/issues/39)  therefore the calico service will not enable Calico on Docker unless you first upgrade Docker _before_ launching Calico service.